app/controllers/welcome_controller.rb

```diff
@@ -1,5 +1,10 @@
 class WelcomeController < ApplicationController
   def index
+    if current_user
```

Comment:
@DarylMcCullough I don't understand what is going on here. Are you not allowing signed in users to see the homepage?